doc/InfiniTimeVision.md

```diff
@@ -0,0 +1,28 @@
+The purpose of this document is to steer efforts towards a common goal, and as such should be taken into consideration with all developments.
```

Comment:
Let's put the `Vision` part as <h1> at the top of the document and make the other headers level 2 <h2>

```suggestion
# InfiniTime Vision

The purpose of this document is to steer efforts towards a common goal, and as such should be taken into consideration with all developments.
```